jbt-it/neomdb
# Created new branch for #135

How to test:
- go to /internes-projekt/5 and check if all the data is displayed correctly
- check if the fields are editable and change a value
- check if the value is actually changed

closes #190
